deepchecks/vision/utils/detection_precision_recall.py

```diff
@@ -0,0 +1,212 @@
+from collections import defaultdict
+
+from ignite.metrics import Metric
+from ignite.metrics.metric import sync_all_reduce, reinit__is_reduced
+import torch
+import numpy as np
+from .iou_utils import compute_ious
+
+
+class DetectionPrecisionRecall(Metric):
```

Comment:
For now should be renamed to AveragePrecision, because this is what this will display until we refactor it to return many values.